src/test/java/com/meetsky/step_definitions/UploadEditDeleteFile_StepDefinitions.java

```diff
@@ -0,0 +1,502 @@
+package com.meetsky.step_definitions;
+
+import com.meetsky.pages.UploadEditDeleteFilePage;
+import com.meetsky.utilities.Driver;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class UploadEditDeleteFile_StepDefinitions {
+
+    UploadEditDeleteFilePage uploadEditDeleteFilePage = new UploadEditDeleteFilePage();
+
+
+    @Given("the user is on the login page")
+    public void theUserIsOnTheLoginPage() {
+        Driver.getDriver().get("https://qa.meetsky.net/index.php/login");
+    }
+
+    @And("the user enters the correct credentials.")
+    public void theUserEntersTheCorrectCredentials() {
+        Driver.getDriver().get("https://qa.meetsky.net/index.php/login");
+        Driver.getDriver().findElement(By.id("user")).sendKeys("Employee21");
+        Driver.getDriver().findElement(By.id("password")).sendKeys("Employee123");
+        Driver.getDriver().findElement(By.id("submit-form")).click();
+        WebDriverWait wait = new WebDriverWait(Driver.getDriver(), 20);
```

Comment:
You can extract the wait object to the class level and use it in everywhere in the class just like the uploadEditDeleteFilePage object. With the help of this you won't need to create same wait object şn every stepdef method.